runtime/tests/test_utils.h

```diff
@@ -40,3 +40,14 @@ double randDouble(bool isNan) {
     assert(isnan(result) == isNan);
     return result;
 }
+
+TaggedPtr getNil() {
+    GC char* ptr = (GC char*)malloc(sizeof(int64_t));
+    ptr = NULL;
+    return ptr;
```

Comment:
If you want to return null pointer, I think you will not need to malloc here.

Reply:
Fixed